.dockerignore

```diff
@@ -0,0 +1,14 @@
+.editorconfig
+.git/
+.github/
+AUTHORS.md
+CONTRIBUTING.md
+LICENSE
+Makefile
+NOTICE
+README.md
+arm/
+powerpc/
+mips/
+build/docker/
+deploy/docker/
```

Comment:
we only have `README.md` and `LICENSE`, we don't have any of the rest. maybe we can exclude `/docs` directory too.